en/micro-integrator/docs/references/mediators/call-Template-Mediator.md

```diff
@@ -145,3 +152,39 @@ dynamically set via the message context.
         </sequence>
     </template>
 ```
+
+### Example 3
+
+Consider an example where the sequence template is configured to log the greeting message that is passed from the mediation sequence in the REST API. According to the sequence template, a value for the greeting message is mandatory. 
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<template name="sequence-temp" xmlns="http://ws.apache.org/ns/synapse">
+    <parameter isMandatory="false" defaultValue="Welcome" name="greeting_message"/>
+    <sequence>
+        <log level="custom">
+            <property expression="$func:greeting_message" name="greeting"/>
+        </log>
+    </sequence>
+</template>
+```
+
+However, this REST API is not passing a greeting message to this template. Also, a <b>default</b> greeting message is not defined in the template. To handle this error <b>at the point of calling the template</b>, you can add the 'onError' parameter to the Call Template mediator and call an error-handling sequence.
```

Comment:
However, in the following example REST API is not passing a greeting message to this template. Also, a <b>default</b> greeting message is not defined in the template. In that case an error will get triggered when calling the template. If you need to handle this error, or in general any error that may occur at execution of the mediation logic inside the template, you can add the...